docs/Projects/TAD/ADRs/0003-tad-tool.md

```diff
@@ -0,0 +1,52 @@
+# ADR-0003 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organizations to get more grip on their algorithms.
+Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a [standardized way](../Reporting-Standard/latest.md).
+
+There are several existing open source
+[tools](../Existing-Tools/Comparison/tools/) which
+we [have assessed](../Existing-Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](../Existing-Tools/Comparison/requirements/) we have.
+
+These are our main requirements:
```

Comment:
Should we add an requirement about multi language? A.o. besides Dutch, that it also supports Fries?

Reply:
It is listed in the other requirements, but to accommodate for it, we could include it also here like:
* The tool supports a multilingual user interface, initially in Dutch and possibly Frysian

Reply:
@uittenbroekrobbert what do you think of replacing `possibly` with `in the future`?

> The tool supports a multilingual user interface, initially in Dutch and in the future also Frisian